interface/breeze-electron/src/components/Vitals/Vitals.jsx

```diff
@@ -55,7 +59,7 @@ export default class Vitals extends React.Component {
   }
 
   toggleMode(value){
-    this.state.mode = modes[(modes.indexOf(value) + 1) % 3];
+    this.state.mode = value;
```

Comment:
should never mutate state directly, change this to this.setState({mode: value})

Reply:
thx! :)